BuildTools/GnupgSigningTool/GnupgSigningTool.csproj

```diff
@@ -7,7 +7,8 @@
     <OutputType>Exe</OutputType>
     <RootNamespace>GnupgSigningTool</RootNamespace>
     <AssemblyName>GnupgSigningTool</AssemblyName>
-    <TargetFrameworkVersion>v4.7</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.6.2</TargetFrameworkVersion>
```

Comment:
Is this actually a downgrade?

Reply:
It is. There's no code which requires 4.7 within it and moving it to 4.6.2 keeps all projects on the same framework.